msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md

```diff
@@ -48,6 +49,15 @@ For more information about the meeting-specific RSC permissions, see [RSC permis
 
 After you've configured the permissions, configure your app to receive [change notifications](/graph/teams-changenotifications-callrecording-and-calltranscript) for transcripts and recordings available events or all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content and recording. Your app can fetch the transcript and recording for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` file. The recording of the meeting is available as an `.mp4` file.
 
+You can use the following types of change notification for your app:
+
+| Use | When... |Supported resource paths|
```

Comment:
Can we consider changing the table headings to | Subscription Scope | Description | Supported resource paths |

Reply:
Updated as suggested